ui/app/pages/swaps/awaiting-swap/awaiting-swap.js

```diff
@@ -70,8 +70,17 @@ export default function AwaitingSwap ({
 
   let feeinFiat
   if (usedQuote && tradeTxParams) {
-    const renderableGasFees = getRenderableGasFeesForQuote(usedQuote.gasEstimateWithRefund || usedQuote.averageGas, approveTxParams?.gas || '0x0', tradeTxParams.gasPrice, currentCurrency, conversionRate)
-    feeinFiat = renderableGasFees.feeinFiat?.slice(1)
+    const renderableNetworkFees = getRenderableNetworkFeesForQuote(
+      usedQuote.gasEstimateWithRefund || usedQuote.averageGas,
+      approveTxParams?.gas || '0x0',
+      tradeTxParams.gasPrice,
+      currentCurrency,
+      conversionRate,
+      tradeTxParams.value,
```

Comment:
Is the expected type for `trade.value` documented anywhere? It's treated as a decimal in one place, but as a hex number in two other places. The only API validation for it is `truthyString`, which doesn't say much.

Reply:
This is addressed in #9730

Reply:
So.... it's a hex number then, or...?